function-sidecar/src/main/java/io/sk8s/sidecar/HttpDispatcher.java

```diff
@@ -16,19 +16,37 @@
 
 package io.sk8s.sidecar;
 
+import org.springframework.retry.backoff.ExponentialBackOffPolicy;
+import org.springframework.retry.policy.SimpleRetryPolicy;
+import org.springframework.retry.support.RetryTemplate;
 import org.springframework.web.client.RestTemplate;
 
+import javax.annotation.PostConstruct;
+
 /**
  * @author Mark Fisher
  */
 public class HttpDispatcher implements Dispatcher {
 
-	private static final String INVOKER_URL = "http://localhost:8080/";
+    private static final String INVOKER_URL = "http://localhost:8080/";
+
+    private final RestTemplate restTemplate = new RestTemplate();
+    private final RetryTemplate retryTemplate = new RetryTemplate();
+    private final ExponentialBackOffPolicy backOffPolicy = new ExponentialBackOffPolicy();
+    private final SimpleRetryPolicy retryPolicy = new SimpleRetryPolicy();
+
+    @Override
+    public String dispatch(String input) {
+        return this.retryTemplate.execute(retryContext -> restTemplate.postForObject(INVOKER_URL, input, String.class));
+    }
 
-	private final RestTemplate restTemplate = new RestTemplate();
+    @PostConstruct
```

Comment:
all of this could be handled in a default constructor for now, and we probably don't need instance vars for anything other than the `retryTemplate` (if we decide to expose these for configurability, we'd probably refactor to have each as a `@Bean`)